_content/tour/por/goroutines.article

```diff
@@ -1,116 +1,68 @@
+O agendador do sistema operacional tem a tarefa de criar a ilusão de que várias tarefas estão sendo executadas ao mesmo tempo, mesmo quando isso é fisicamente impossível. Isso requer algumas compensações no design do agendador. Antes de prosseguir, é importante definir algumas palavras-chave.
 
-*Work:* A set of instructions to be executed for a running application. This is
-accomplished by threads and an application can have 1 to many threads.
+*Work:* Um conjunto de instruções a serem executadas por uma aplicação em execução. Isso é realizado por meio de threads, e uma aplicação pode ter de 1 a várias threads.
 
-*Thread:* A path of execution that is scheduled and performed. Threads are responsible
-for the execution of instructions on the hardware.
+*Thread:* Um caminho de execução que é agendado e executado. As threads são responsáveis pela execução de instruções no hardware.
 
-*Thread* *States:* A thread can be in one of three states: Running, Runnable, or
-Waiting. Running means the thread is executing its assigned instructions on the
-hardware by having a G placed on the M. Runnable means the thread wants time on
-the hardware to execute its assigned instructions and is sitting in a run queue.
-Waiting means the thread is waiting for something before it can resume its work.
-Waiting threads are not a concern of the scheduler.
+ *Estados* *da* *Thread:* Uma thread pode estar em um dos três estados: Executando (Running), Pronta para Executar (Runnable) ou Aguardando (Waiting). Executando significa que a thread está executando suas instruções atribuídas no hardware, tendo uma G associada a um M. Pronta para Executar significa que a thread deseja tempo no hardware para executar suas instruções atribuídas e está na fila de execução (run queue). Aguardando significa que a thread está aguardando algo antes de poder retomar seu trabalho. Threads em estado de espera não são uma preocupação do agendador.
 
-*Concurrency:* This means undefined out of order execution. In other words, given
-a set of instructions that would be executed in the order provided, they are executed
-in a different undefined order, but all executed. The key is, the result of executing
-the full set of instructions in any undefined order produces the same result. You will
-say work can be done concurrently when the order the work is executed in doesn’t matter,
-as long as all the work is completed.
+*Concorrência:* Isso significa uma execução indefinida e fora de ordem. Em outras palavras, dado um conjunto de instruções que seriam executadas na ordem fornecida, elas são executadas em uma ordem indefinida diferente, mas todas são executadas. A chave é que o resultado da execução do conjunto completo de instruções em qualquer ordem indefinida produz o mesmo resultado. Pode-se dizer que o trabalho pode ser feito de forma concorrente quando a ordem em que o trabalho é executado não importa, contanto que todo o trabalho seja concluído.
```

Comment:
I think here would fit better "indefinidamente fora de ordem". What I understood was: the execution could be out of order and in an undefined order, but it could also be in the "normal" order.

But I'm not sure, my english isn't that good.

Reply:
I believe that's fine